File: RECURSION/QUESTION 2/Lazysequence.java
import java.util.Scanner;


public class Lazysequence {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.close();
        lazySequence(n);
    }

    public static void lazySequence(int n) {
        if (n == 0) {
            System.out.print(n + " ");
            return;
        }
        System.out.print(n + " ");
        lazySequence(n - 1);
        System.out.print(n + " ");
    }

}

File: RECURSION/QUESTION 2/Lazysequence.c
#include<stdio.h>

void lazysequence(int n)
{
    if(n==0)
    {
        printf("%d ",n);
        return;
    }
        printf("%d ",n);
        lazysequence(n-1);
        printf("%d ",n);

}

int main()
{
    int n;
    scanf("%d",n);
    lazysequence(n);
}
